src/Adapter/AbstractAdapter.php

```diff
@@ -97,10 +98,25 @@ public function resetFilter($filterName)
     /**
      * {@inheritdoc}
      */
-    public function resetOperationsFilter($filterName)
+    public function resetOperationsFilter($filterName, $value = null)
     {
         if ($this->operationsFilters->offsetExists($filterName)) {
-            $this->operationsFilters->offsetUnset($filterName);
+            if ($value === null) {
+                $this->operationsFilters->offsetUnset($filterName);
+            } else {
+                $operations = $this->operationsFilters->offsetGet($filterName);
+                foreach ($operations as $filterId => $filterValue) {
```

Comment:
May be more clean

```
if (isset($operations[$value])) {
   unset($operations[$value]);
}
```

Reply:
I want to check strictly on this index ^^

Reply:
Oh thanks, Didn't see (came from test I think :sweat_smile: )

Reply:
:ok_hand: